### ISSUE TITLE
Use Inline Images to Ensure Display in High Contrast Modes (Recommendation)

### ISSUE THREAD

**Issue by jscholes** [benetech/ImageShare#52]
Use inline images to ensure that the content will be visible in high contrast modes, such as the one available on Windows 10.  This is opposed to other techniques such as CSS background images which will be removed from the page when such a mode is enabled.  For example, the Explore Collections images, Imageshare and Benetech logos, search magnifying glass and similar icons, Advanced Search downward facing arrow, social icons, filter “X” close icon and any content resource images should all be marked up inline.

Note: certain CSS media queries, such as `-ms-high-contrast`, are available for situations where particular techniques should be used to target high contrast users.  These can be discussed on a case-by-case basis.

**Comment by sinabahram** [comment]
@jkva  now that we have a preliminary implementation, can you check this?

**Comment by jkva** [comment]
Closing, as not using background images anywhere except page header.
